tap-snapshots/test/lib/commands/profile.js.test.cjs

```diff
@@ -47,56 +47,56 @@ github	https://github.com/npm
 `
 
 exports[`test/lib/commands/profile.js TAP profile get no args default output > should output table with contents 1`] = `
-name: foo
-email: foo@github.com (verified)
-two-factor auth: auth-and-writes
-fullname: Foo Bar
-homepage: https://github.com
-freenode: foobar
-twitter: https://twitter.com/npmjs
-github: https://github.com/npm
-created: 2015-02-26T01:26:37.384Z
-updated: 2020-08-12T16:19:35.326Z
+[1mname[22m: foo
+[1memail[22m: foo@github.com (verified)
+[1mtwo-factor auth[22m: auth-and-writes
+[1mfullname[22m: Foo Bar
+[1mhomepage[22m: https://github.com
+[1mfreenode[22m: foobar
+[1mtwitter[22m: https://twitter.com/npmjs
+[1mgithub[22m: https://github.com/npm
+[1mcreated[22m: 2015-02-26T01:26:37.384Z
+[1mupdated[22m: 2020-08-12T16:19:35.326Z
```

Comment:
maybe add a mock for `chalk` the same way `ansistyle` was mocked so that we don't need to shuffle all these snapshots

Reply:
We have lots of other tests that snapshot colors, this brings these snapshots in line with the others.

Reply:
The mock for chalk is masking a whole other issue in that we can't remove colors from tap output itself right now which is a whole other ball of wax.